docs/install-config/reconfigure-manage-lifecycle.md

```diff
@@ -63,10 +63,10 @@ To reconfigure Harbor, perform the following steps.
     sudo prepare
     ```
 
-    To reconfigure Harbor to install Notary, Clair, and the chart repository service, include all of the components in the `prepare` command.
+    To reconfigure Harbor to install Notary and the chart repository service, include all of the components in the `prepare` command.
 
     ```sh
-    sudo prepare --with-notary --with-clair --with-chartmuseum
+    sudo prepare --with-notary --with-chartmuseum
```

Comment:
wait..do we not have an option to install with trivy?

Reply:
i updated this to say --with-trivy